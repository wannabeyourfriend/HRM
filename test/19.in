3
2
inbox
jump 1