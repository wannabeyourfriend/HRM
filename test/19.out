Fail with execution times 17